packages/config/config/devices/0x027a/zen52.json

```diff
@@ -0,0 +1,125 @@
+{
+	"manufacturer": "Zooz",
+	"manufacturerId": "0x027a",
+	"label": "ZEN52",
+	"description": "Z-Wave Plus 700 Series Double Relay",
+	"devices": [
+		{
+			"productType": "0x0104",
+			"productId": "0x0202"
```

Comment:
FWIW [there is now a ZWave Alliance ID of 4609](https://products.z-wavealliance.org/products/4609)

Reply:
```suggestion
			"productId": "0x0202",
			"zwaveAllianceId": 4609
```